x-pack/plugins/maps/public/classes/styles/vector/components/legend/style_error.tsx

```diff
@@ -9,11 +9,10 @@ import React, { useEffect, useState } from 'react';
 import { EuiCallOut, EuiFlexGroup, EuiFlexItem, EuiText, EuiToolTip } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { DynamicStyleProperty } from '../../properties/dynamic_style_property';
-import { DataRequest } from '../../../../util/data_request';
 
 interface Props {
   error: Error;
-  style: DynamicStyleProperty<DataRequest>;
+  style: DynamicStyleProperty<any>;
```

Comment:
Lets no use `any`. The type defaults to object so you should be able to use only `DynamicStyleProperty`.